Python/initconfig.c

```diff
@@ -2222,6 +2224,11 @@ config_usage(int error, const wchar_t* program)
     }
 }
 
+static void
+config_xoptions_usage()
+{
+    fputs(usage_xoptions, stdout);
+}
```

Comment:
Maybe this does not need to be a function.  I started by copying config_usage but ended up with a single line 🙂

Reply:
Invalid `-X` option output could be improved, although I'm just not sure if it's worth additional complexity:

```
% ./python.exe -X foo
Fatal Python error: Unknown value for option -X (see -X help)
Python runtime state: preinitialized
```

Reply:
Is your example output here what you would like?  Because the PR does add `(see -X help)` 🙂

Reply:
It might be nicer to mimic what other invalid options print, e.g.:

```
% ./python.exe -X foo
Unknown option: -X foo
usage: (null) [option] ... [-c cmd | -m mod | file | -] [arg] ...
Try `python -h' for more information.
```

although maybe `Try python -X help for more information`?

Reply:
Because it’s a fatal error, I am unsure about changing the string format.
Keeping two lines with `Fatal Python error: details details` and `Python runtime state: info` seems better.

Reply:
Why would `-X foo` be a fatal error but not `--foo`?  IOW, it feels inconsistent, but maybe this isn't important enough to increase complexity.

Reply:
Probably because some X options change fundamental behaviour, so they are enabled during early startup and cause fatal errors.

Reply:
> Probably because some X options change fundamental behaviour, so they are enabled during early startup and cause fatal errors.

Yep, in which case, let's just leave it as you have it.